packaging/opensuse/snapd.spec

```diff
@@ -51,29 +56,18 @@
 # Additional entry of $GOPATH during the build process.
 # This is designed to be a sub-directory of {_builddir}/{name}-{version}
 # because that directory is automatically cleaned-up by the build process.
-%global         indigo_gopath   %{_builddir}/%{name}-%{version}/gopath
+%global indigo_gopath   %{_builddir}/%{name}-%{version}/gopath
 
 # Directory where "name-version" directory from upstream taball is unpacked to.
 # This directory is arranged so that it is already contained inside the future
 # GOPATH so that nothing needs to be moved or copied for "go build" to work.
-%global         indigo_srcdir   %{indigo_gopath}/src/%{import_path}
-
-%global with_test_keys  0
-
-%if %{with testkeys}
-%global with_test_keys 1
-%else
-%global with_test_keys 0
-%endif
+%global indigo_srcdir   %{indigo_gopath}/src/%{import_path}
```

Comment:
Same here?

Reply:
Indigo may provide this value. Let's see how things evolve. I would keep it because it works.